common/test/db_cache/bok_choy_schema.sql

```diff
@@ -21,61 +21,61 @@ CREATE TABLE `api_manager_coursecontentgrouprelationship` (
   `record_active` tinyint(1) NOT NULL,
   `group_profile_id` int(11) NOT NULL,
   PRIMARY KEY (`id`),
-  UNIQUE KEY `api_manager_coursecontentgroupre_course_id_44c879ae670afc9_uniq` (`course_id`,`content_id`,`group_profile_id`),
-  KEY `api_manager_coursecontentgrouprelati_course_id_7bf540377b22727d` (`course_id`,`content_id`),
-  KEY `api_manager_coursecontentgrouprelationship_ff48d8e5` (`course_id`),
-  KEY `api_manager_coursecontentgrouprelationship_cc8ff3c` (`content_id`),
-  KEY `api_manager_coursecontentgrouprelationship_e1ffc2ea` (`group_profile_id`),
+  UNIQUE KEY `edx_solutions_api_integration_coursecontentgroupre_course_id_44c879ae670afc9_uniq` (`course_id`,`content_id`,`group_profile_id`),
+  KEY `edx_solutions_api_integration_coursecontentgrouprelati_course_id_7bf540377b22727d` (`course_id`,`content_id`),
```

Comment:
I think we should not update reference in this file. This is temporary file which can regenerated with the updated tables.  If we modify reference here we would not have corresponding changes in DB for instance we would not have this index in DB `edx_solutions_api_integration_coursecontentgrouprelati_course_id_7bf540377b22727d`.